lib/kvdb/ikvdb.c

```diff
@@ -479,6 +479,31 @@ ikvdb_drop(const char *const kvdb_home)
     return err;
 }
 
+merr_t
+ikvdb_mclass_info_get(
+    struct ikvdb *const           kvdb,
+    const char *                  mclass,
+    struct hse_mclass_info *const info)
+{
+    enum mpool_mclass  mc;
+    struct ikvdb_impl *self;
+
+    INVARIANT(kvdb);
+    INVARIANT(info);
+
+    self = ikvdb_h2r(kvdb);
+
+    if (!strcmp(mclass, HSE_MCLASS_CAPACITY_NAME)) {
```

Comment:
I would put this in a for loop and use mpool_mclass_to_string[]. That will save me from updating this function for pmem mclass.